apollo-environment/src/main/java/com/spotify/apollo/environment/ApolloEnvironmentModule.java

```diff
@@ -65,8 +65,14 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(ApolloEnvironmentModule.class);
 
-  public static ApolloEnvironmentModule create() {
```

Comment:
I've written (and others i know of) command line tools that use the environment module to get a configured Apollo Client. But now as I'm writing this, I realise I could just use the client modules directly.

So it's probably ok :)